src/main/scala/is/hail/methods/LinearRegressionBurden.scala

```diff
@@ -66,24 +64,18 @@ object LinearRegressionBurden {
 
     val linregRDD = sampleKT.mapAnnotations { keyedRow =>
       val key = keyedRow.get(0)
-
-      RegressionUtils.statsKeyedRow(keyedRow, y) match {
-        case Some((x, xx, xy)) =>
-          val qtx = QtBc.value * x
-          val qty = QtyBc.value
-          val xxp: Double = xx - (qtx dot qtx)
-          val xyp: Double = xy - (qtx dot qty)
-          val yyp: Double = yypBc.value
-
-          val b = xyp / xxp
-          val se = math.sqrt((yyp / xxp - b * b) / d)
-          val t = b / se
-          val p = 2 * T.cumulative(-math.abs(t), d, true, false)
-
-          Row(key, b, se, t, p)
-        case None =>
-          Row(key +: emptyStats)
-      }
+      val x = RegressionUtils.keyedRowToVectorDouble(keyedRow)
+      val qtx = QtBc.value * x
+      val xxp: Double = (x dot x) - (qtx dot qtx)
+      val xyp: Double = (x dot y) - (qtx dot QtyBc.value)
+      val yyp: Double = yypBc.value
+
+      val b = xyp / xxp
+      val se = math.sqrt((yyp / xxp - b * b) / d)
+      val t = b / se
+      val p = 2 * T.cumulative(-math.abs(t), d, true, false)
+
+      Row(key, b, se, t, p)
```

Comment:
I'm not sure why this is duplicated.  Have a function that takes a vector and returns the result of linear regression as an annotation.  Then you can use it here and in linreg.